export/serializers/csv_serializer.py

```diff
@@ -21,14 +21,21 @@
 import codecs
 import csv
 import re
-import StringIO
+try:
+    import StringIO
+except ImportError:
+    import io as StringIO
```

Comment:
A matter of style: Since StringIO.StringIO is the only part being used, I would rather do:
try:
    from io import StringIO
except ImportError:
    from StringIO import StringIO

and then replace the StringIO.StringIO bits later in the file with just StringIO.